torch/csrc/distributed/rpc/init.cpp

```diff
@@ -405,6 +406,42 @@ If the future completes with an error, an exception is thrown.
                   >>> # The inserted callback will print the return value when
                   >>> # receiving the response from "worker1"
                   >>> fut.add_done_callback(callback)
+          )")
+      .def(
+          "set_result",
```

Comment:
Is it an explicit goal to mimic `concurrent.Futures` API? If so, should we have `get()` as well?

Reply:
Yes, according to the discussion in #36071, Python users might be familiar with this set of API. Let me add that. You mean `Future.result()` instead of `Future.get()`?

Reply:
Hmm, this will then be duplicated with the existing `wait` API. Let's postpone that until we discussed this with the team.